apps/demo/emails/codepen-challengers.tsx

```diff
@@ -0,0 +1,407 @@
+import {
+  Body,
+  Button,
+  Container,
+  Head,
+  Heading,
+  Column,
+  Html,
+  Img,
+  Link,
+  Preview,
+  Section,
+  Text,
+} from '@react-email/components';
+import * as React from 'react';
+
+const baseUrl = process.env.VERCEL_URL
+  ? `https://${process.env.VERCEL_URL}`
+  : '';
+
+export default function Email() {
+  return (
+    <Html>
+      <Head />
+      <Preview>#CodePenChallenge: Cubes</Preview>
+      <Body style={main}>
+        <Section style={header}>
+          <Container style={containerHeader}>
+            <Img
+              src={`${baseUrl}/static/codepen-challengers.png`}
+              width={600}
+              alt="codepen"
+            />
+          </Container>
+        </Section>
+        <Container style={container}>
```

Comment:
The structure should be `Container (only 1) > Section > Row > Column`. So having `Section > Container` is not the recommended way to go.

Reply:
Thank you for you review, I will look for another solution for this.

Reply:
@bukinoshita could you see if it turned out better that way?

Reply:
Looks much better, just ran a few tests and the only thing that needs to change is the padding on the Button to work on Outlook.

Instead of using the `padding` style, you need to remove that and pass in the component as `<Button style={buttonStyle} pY={15} pX={30}>` for example.

Reply:
I don't know if this is a bug, but I'm trying to use the pX and pY props and it looks like the component has duplicated the style in the <a> and <span> tags causing the border to be duplicated, like this

![image](https://user-images.githubusercontent.com/19210426/221443182-beb3edbd-eff2-4af6-babb-ad6ea22aaff8.png)

Reply:
Hm, I see. This is caused by https://github.com/resendlabs/react-email/issues/505. Let's revert and then fix the outlook later.

Reply:
Ok! I will remove the border

Reply:
@bukinoshita done!

Reply:
Thanks @bruno88cabral